src/libstd/process.rs

```diff
@@ -1124,7 +1124,14 @@ pub fn exit(code: i32) -> ! {
 ///
 /// Note that because this function never returns, and that it terminates the
 /// process, no destructors on the current stack or any other thread's stack
-/// will be run. If a clean shutdown is needed it is recommended to only call
+/// will be run.
+///
+/// This is in contrast to the default behaviour of [`panic!`] which unwinds
+/// the current thread's stack and calls all destructors.
+/// When `panic="abort"` is set, either as an argument to `rustc` or in a
+/// crate's Cargo.toml, [`panic!`] and `abort` are equivalent.
```

Comment:
Is this correct? With panic=abort, I assumed a settable panic handler and/or a backtrace would happen, whereas process::abort will not call a panic handler.

Reply:
Just tested this and it looks like you're right - I'd missed that. I'll change the wording and mentioned that process::abort skips the panic handler.